contrib/IECoreUSD/src/IECoreUSD/ShaderAlgo.cpp

```diff
@@ -309,15 +309,33 @@ pxr::UsdShadeConnectableAPI createShaderPrim( const IECoreScene::Shader *shader,
 	{
 		throw IECore::Exception( "Could not create shader at " + path.GetAsString() );
 	}
+
 	const std::string type = shader->getType();
+
 	std::string typePrefix;
-	size_t typeColonPos = type.find( ":" );
-	if( typeColonPos != std::string::npos )
+	if( boost::starts_with( shader->getName(), "Pxr" ) || boost::starts_with( shader->getName(), "Lama" ) )
 	{
-		typePrefix = type.substr( 0, typeColonPos ) + ":";
-		if( typePrefix == "ai:" )
+		// Leave the type prefix empty. This should be the default, but we are currently only doing this
+		// for a small number of shaders that we can be completely confident require it, in order to
+		// preserve backwards compatibility.
+	}
+	else
+	{
+		size_t typeColonPos = type.find( ":" );
+		if( typeColonPos != std::string::npos )
 		{
-			typePrefix = "arnold:";
+			// According to our current understanding, this is almost completely wrong. Renderer's like
```

Comment:
This would suggest that it works for Cycles, although it's not clear if it's what they prefer or something they tolerate : https://github.com/blender/cycles/blob/main/src/hydra/material.cpp#L435-L437. So I think "almost completely wrong" is a bit strong...

Reply:
Yeah, we'll definitely need to do something with the OSL stuff, but I think "almost completely wrong" is overstating it.